xbmc/utils/SortUtils.h

```diff
@@ -166,6 +166,8 @@ typedef enum
   SortByOriginalTitle,
   /// __58__ : Sort by provider                   <em>(String: <b><c>Provider</c></b>)</em>
   SortByProvider,
+  /// __59__ : Sort by user preference            <em>(String: <b><c>UserPreference</c></b>)</em>
+  SortByUserPreference,
```

Comment:
Sure, please tell e how to do this.

Reply:
```
diff --git a/xbmc/utils/SortUtils.h b/xbmc/utils/SortUtils.h
index 7f64089a99..91c2e32604 100644
--- a/xbmc/utils/SortUtils.h
+++ b/xbmc/utils/SortUtils.h
@@ -167,6 +167,7 @@ typedef enum
   /// __58__ : Sort by provider                   <em>(String: <b><c>Provider</c></b>)</em>
   SortByProvider,
   /// __59__ : Sort by user preference            <em>(String: <b><c>UserPreference</c></b>)</em>
+  /// @skinning_v20 <b>SortByUserPreference</b> New sort method added.
   SortByUserPreference,
 } SortBy;
 ///@}
```

This will do:

![image](https://user-images.githubusercontent.com/7375276/195053080-7eb60556-0733-42b7-9e96-a756ae6af271.png)

Reply:
Thanks, will add this.

Reply:
I should add this also for "SortByProvider", which was also introduced for v20.